### DISCUSSION TITLE
Adhesion gel

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4121] [Feature Suggestions]
In old aperture (condemied) i seen the green gel tube.

I think will be cool to add adhesion gel to bee mod like the reflection gel.

**Comment by vrad-exe** [comment]
Adhesion Gel was never fully implemented in any released version of the game, and the tricks used to make it functional don't work in the Puzzlemaker. See the [FAQ question](https://github.com/BEEmod/BEE2-items/wiki/FAQ#can-you-add-adhesion-gelsome-other-custom-gel-to-bee24).

**Reply by vrad-exe** [reply]
It's neat that you consider adhesion gel to be green though lol
